Description: Allow python-oslo.serialization older than 1.2.0
 We have an older version than 1.2.0 on Debian. That's the reason for this change.
 Same applies to python-oslo.utils.
 .
Author: gustavo panizzo <gfa@zumbi.com.ar>

---
Origin: vendor
Forwarded: not-needed
Reviewed-By: gustavo panizzo <gfa@zumbi.com.ar>
Last-Update: <2015-01-27>

--- a/requirements.txt
+++ b/requirements.txt
@@ -9,8 +9,8 @@
 cliff-tablib>=1.0
 oslo.config>=1.6.0  # Apache-2.0
 oslo.i18n>=1.0.0  # Apache-2.0
-oslo.utils>=1.2.0                       # Apache-2.0
-oslo.serialization>=1.2.0               # Apache-2.0
+oslo.utils>=1.0.0                       # Apache-2.0
+oslo.serialization>=1.0.0               # Apache-2.0
 python-glanceclient>=0.15.0
 python-keystoneclient>=0.11.1
 python-novaclient>=2.18.0
